src/components/common/Certify.vue

```diff
@@ -174,4 +174,14 @@ export default class Certify extends Vue {
     margin-top: -1.25rem;
   }
 }
+
+:deep() {
+  .v-input--selection-controls__ripple {
```

Comment:
Had to configure this manually from scratch.
Before:
![Screenshot (112)](https://github.com/bcgov/business-filings-ui/assets/122301442/180b1280-5a82-480a-a52d-727494809e7a)

After:
![Screenshot (111)](https://github.com/bcgov/business-filings-ui/assets/122301442/847b3a70-0cad-4374-ae38-820cdd84bcbb)

Reply:
Your change makes this checkbox behave differently from other checkboxes. Why not do something like this?

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/3a2bea21-2915-440c-9184-e0252a1dfbc4)

Reply:
Yup, I thought the circle thing was wrong. It's just the location of the ripple was wrong. I was thinking of something else.

After latest commit:
![Screenshot (113)](https://github.com/bcgov/business-filings-ui/assets/122301442/a3258cae-7992-47c7-9e21-af66ade35268)

Reply:
I'm ok to resolve this conversation.